docker/owlbot/nodejs/entrypoint.sh

```diff
@@ -14,8 +14,7 @@
 # limitations under the License.
 
 set -e
-# Doesn't get sourced in a login shell, according to bash's man page and confirmed
-# via experiment, so we have to source it manually.
-source "$HOME/.bashrc"
+source /root/.nvm/nvm.sh
```

Comment:
@SurferJeffAtGoogle it took some trial and error to figure out an appropriate `entrypoint.sh` when `dir` was set in Cloud Build, this approach is working, see:

https://github.com/bcoe/owlbot-testing/pull/14

Reply:
Was it failing without this change?

Reply:
It was failing with `source "$HOME/.bashrc"`, I believe because `$HOME` becomes `dir`.

It was also failing unless I sourced `source /root/.nvm/nvm.sh`, because of not being able to find the `npm` bin.